src/plugins/geomap/_base.scss

```diff
@@ -23,8 +23,6 @@
 }
 
 .ol-popup {
-	-moz-box-shadow: 0 1px 4px rgba(0,0,0,.2);
```

Comment:
These `-moz-box-shadow` values are the only ones that don't get added back, since they were replaced by the native/-webkit versions for some time https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#browser_compatibility